massgov/mayflower
# React/DP-14436:  Update storybook to version 5.1.8.

## Description
This updates storybook to its latest version and fixes bugs in ImagePromo's story.

Addon-info seems to break with the 5.1.8 release related to using markdown files. This has been fixed in at least version 5.2.0-alpha.30 however. See this ticket for more information: https://github.com/storybookjs/storybook/issues/6981

We should upgrade to 5.2.0 when it's released. For now, using the alpha version for only addon-info fixes the markdown bug. All the other storybook packages are set to 5.1.8.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-14436)

## Steps to Test


1. Pull the branch and run `npm install`. Verify that all tests pass for backstop. Verify that production and develop builds run with no issues.
